config/routes.rb

```diff
@@ -22,12 +21,14 @@
     post '/sign_up/register' => 'sign_up/registrations#create', as: :sign_up_register
     get '/sign_up/start' => 'sign_up/registrations#show', as: :sign_up_start
     get '/sign_up/verify_email' => 'sign_up/emails#show', as: :sign_up_verify_email
+    get '/sign_up/create_password' => 'sign_up/confirmations#show', as: :user_confirmation
+    get '/sign_up/enter_email_again' => 'sign_up/confirmations#new', as: :new_user_confirmation
+    post '/sign_up/register_again' => 'sign_up/confirmations#create', as: :sign_up_register_again
+    patch '/sign_up/create_password' => 'sign_up/confirmations#confirm'
```

Comment:
I can try! these are the default routes: https://github.com/plataformatec/devise/blob/master/lib/devise/rails/routes.rb#L59

Looks like we invented this one ourselves, so can probably improve somewhat easily. will report back